tests/ensure_provider_tests.py

```diff
@@ -72,11 +72,17 @@
     "playbooks/tests_team.yml": {
```

Comment:
The commit message does not seem to be correct English. How about "Test installation of NetworkManager plugins"?

Reply:
updated, thanks